docs/pages/management/join-services-to-your-cluster/azure.mdx

```diff
@@ -1,53 +1,42 @@
 ---
-title: Joining Nodes via Azure Managed Identity
-description: Use the Azure join method to add Nodes to your Teleport cluster on Azure
+title: Joining Teleport Services via Azure Managed Identity
```

Comment:
The other pages adjusted here don't include "Teleport" in the title. Not sure which is better, but noting for consistency.